recipes/av2/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "av2" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/argoai/argoverse2-api/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 853aca3c9dd561f189b6c808f3d73ea816fc5e2cce7d98fc3c832dd181c4ef45
+
+build:
+  noarch: python
+  skip: true
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - setuptools >=42
+    - wheel
+
+  host:
+    - numpy
```

Comment:
Is this really needed? Because I don't see anything compiled or built against it. If that's the case, we might need to switch back to arch.

Reply:
Thanks, I've removed it.